modules/custom/dkan_harvest/src/Drush/Commands.php

```diff
@@ -10,7 +10,7 @@
 use Drush\Commands\DrushCommands;
 
 /**
- *
+ * @codeCoverageIgnore
```

Comment:
This one is for my sanity. If I start adding one unit test, I'll have to add all the unit tests.